packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -315,8 +246,109 @@ import { name, version } from '../package'
 
   const startTime = Date.now()
 
+  // Engine check Listr. Separate Listr to avoid https://github.com/cenk1cenk2/listr2/issues/296
+  // Boolean flag
+  let hasPassedEngineCheck = null
+  // Array of strings
+  let engineErrorLog = []
+  // Docs link for engine errors
+  const engineErrorDocsLink = terminalLink(
+    'Tutorial - Prerequisites',
+    'https://redwoodjs.com/docs/tutorial/chapter1/prerequisites'
+  )
+
+  await new Listr(
+    [
+      {
+        title: 'Checking node and yarn compatibility',
+        skip: () => {
+          if (yarnInstall === false) {
+            return 'Warning: skipping check on request'
+          }
+        },
+        task: () => {
+          return new Promise((resolve) => {
+            const { engines } = require(path.join(templateDir, 'package.json'))
+
+            // this checks all engine requirements, including Node.js and Yarn
+            checkNodeVersion(engines, (_error, result) => {
+              if (result.isSatisfied) {
+                hasPassedEngineCheck = true
+                return resolve()
+              }
+              const logStatements = Object.keys(result.versions)
+                .filter((name) => !result.versions[name].isSatisfied)
+                .map((name) => {
+                  const { version, wanted } = result.versions[name]
+                  return style.error(
+                    `${name} ${wanted} required, but you have ${version}`
+                  )
+                })
+              engineErrorLog = logStatements
+              hasPassedEngineCheck = false
+              return resolve()
+            })
+          })
+        },
+      },
+    ],
+    { rendererOptions: { clearOutput: true } }
+  ).run()
+
+  // Show a success message if required engines are present
+  if (hasPassedEngineCheck === true) {
+    console.log(style.success(`✔️ Compatability checks passed.`))
+  }
+
+  // Show an error and prompt if failed engines check
+  if (hasPassedEngineCheck === false) {
+    console.log(style.error(`✖️ Compatability checks failed.`))
+    console.log(`${engineErrorLog.join('\n')}`)
+    console.log(
+      style.error(
+        `\nThis might make your RedwoodJS project incompatible with some deploy targets.`
+      )
+    )
+    console.log(style.header(`\nRelated documentation:`))
+    console.log(style.warning(`${engineErrorDocsLink}\n`))
+    // Prompt user for how to proceed
+    const response = await prompt({
+      type: 'select',
+      name: 'override-engine-error',
+      message: 'How would you like to proceed?',
+      choices: ['Override error and continue install', 'Quit install'],
+      initial: 0,
```

Comment:
Without this line, quitting the compatibility checks prompt (via <kbd>Ctrl</kbd>+<kbd>c</kbd>) logs out something from Node.js core:

```
✖ Compatibility checks failed
  ⚠ node >=14.19.0 <17.0.0 required, but you have 18.9.1

    This may make your project incompatible with some deploy targets
    See Tutorial - Prerequisites (​https://redwoodjs.com/docs/tutorial/chapter1/prerequisites​)

✖ How would you like to proceed? · 
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

Node.js v18.9.1
``` 

```suggestion
      initial: 0,
      onCancel: () => process.exit(1),
```